Tools/ardupilotwaf/boards.py

```diff
@@ -192,7 +198,7 @@ def configure_env(self, cfg, env):
             'm',
         ]
 
-        cfg.check_librt(env)
+        cfg.check_librt(env, mandatory=False)
```

Comment:
In other words: for Linux / cygwin it's mandatory to have clock_gettime(). For MacOS it uses whatever function MacOS provides. We could check for it's availability or write a compat clock_gettime in the missing directory, but I don't think it's worth the effort here.